src/GovUk.Education.ExploreEducationStatistics.Data.Api/Services/PermalinkCsvMetaService.cs

```diff
@@ -94,6 +94,7 @@ public class PermalinkCsvMetaService : IPermalinkCsvMetaService
                 .SingleAsync(
                     predicate: rf =>
                         rf.File.SubjectId == releaseSubject.SubjectId
+                        && rf.File.Type == FileType.Data
```

Comment:
Might be worth having a test with a data and meta file to check that this doesn't regress?

Reply:
Good point, added one extra test `GetCsvMeta_SubjectExists_MultipleReleaseFilesExist` which sets up the metadata file and also a data file for a different subject.